source/assets/sass/components/_callouts.scss

```diff
@@ -19,20 +19,20 @@
   }
 
   &--function {
-    padding-bottom: 0.25rem;
+    padding-bottom: var(--gutter-quarter);
     padding-top: 0;
 
     .signature {
       // Make sure permalinks are visible.
       overflow: visible;
-      margin-left: -1rem;
-      margin-right: -1rem;
+      margin-left: calc(var(--gutter) * -1);
+      margin-right: calc(var(--gutter) * -1);
```

Comment:
@mirisuzanne this site often uses the negative values of these variables. Should we make a negative version in the config/scale file once and use it instead of adding all these calc * -1?

Reply:
I don't have a strong feeling. I'd say either way we should keep the `calc()` intact - but that could either be moved inside a property (eg `--gutter-negative`) or even a function like `negative-var(--gutter)`? The downside of the new property is just that you'd have to define another one every time you need the negative of another size. But happy for you to make the call on that.

Reply:
@mirisuzanne  Since there are only 4 numbers we needed now as their negative equivalent, I added them manually. I also added a card incase we want to make this easier to apply to any number in the future and avoid needing calc.